starfish/test/full_pipelines/api/test_merfish.py

```diff
@@ -248,4 +248,4 @@ def test_merfish_pipeline_cropped_data():
 
     corrcoef = np.corrcoef(tmp[:, 1], tmp[:, 0])[0, 1]
 
-    assert np.round(corrcoef, 4) == 0.9647
+    assert np.round(corrcoef, 4) == 0.9963
```

Comment:
@ambrosejcarr i think @shanaxel42 finally got to the bottom of why Starfish was 'under-counting' relative to the original MERFISH results.